command/agent/consul/check.go

```diff
@@ -47,6 +47,12 @@ func (r *CheckRunner) Start() {
 	r.started = true
 }
 
+func (r *CheckRunner) Started() bool {
```

Comment:
Do you think the following will be ok?
// Started is used to check whether Start() method called for this object
excuse for my English